### PR TITLE
Revert "Updated"

### PR DESCRIPTION
**Pull request by ACNHubert**
Reverts ACNHubert/FutureWeatherForecast#1

### CODE REVIEW

File: app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt [hunk 1]
```diff
@@ -52,15 +52,15 @@ class MainActivity : AppCompatActivity() {
...
-    }
 
     @RequiresApi(Build.VERSION_CODES.N)
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         bindind = DataBindingUtil.setContentView(this, R.layout.activity_main)
 
+
+
+
```
Comment (jobert-beltran) [inline app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt:63]: minor: dont leave white spaces. you can also enable auto-format,auto-organize imports before commit in android studio to keep your code clean ;)

File: app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt [hunk 2]
```diff
@@ -401,28 +397,23 @@ class MainActivity : AppCompatActivity() {
                         editor.putString(Constants.WEATHER_RESPONSE_DATA, weatherResponseJsonString)
                         editor.apply()
 
-//                        var location : String = weatherList.sys.country
-//                        if (location == "PH") {
+                        var location : String = weatherList.sys.country
+                        if (location == "PH") {
```
Comment (jobert-beltran) [inline app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt:401]: why do you filter other countries except PH?

File: app/src/main/res/layout/list_item.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/clListItem"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="16dp"
+    android:background="#80000000">
+
+    <TextView
+        android:id="@+id/textView_temperature"
```
Comment (jobert-beltran) [inline app/src/main/res/layout/list_item.xml:11]: please check proper naming convention for resource ids :)

File: app/src/main/res/layout/list_item.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="25.25 C"
```
Comment (jobert-beltran) [inline app/src/main/res/layout/list_item.xml:17]: never hardcode displayed texts in layout files or other classes. prefer to use string resources :)

https://developer.android.com/guide/topics/resources/string-resource

**Comment by jobert-beltran** [comment]
when raising a PR. add a good title and description of your changes :)

**Comment by jobert-beltran** [comment]
tip: anything under .idea, .gradle, build, app/build you dont have to commit. alternatively you can use this gitignore file to ignore files you should not commit https://github.com/github/gitignore/blob/main/Android.gitignore :)

File: app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt (hunk 1 above)
Comment (ACNHubert) [reply app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt:63]: Okay this is noted!

File: app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt (hunk 2 above)
Comment (ACNHubert) [reply app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt:401]: I tried to filter City name valid for Philippines only. I forgot to remove the Code before I commit.

File: app/src/main/res/layout/list_item.xml (hunk 3 above)
Comment (ACNHubert) [reply app/src/main/res/layout/list_item.xml:11]: Okay noted!

File: app/src/main/res/layout/list_item.xml (hunk 4 above)
Comment (ACNHubert) [reply app/src/main/res/layout/list_item.xml:17]: This is noted, I will add String resource for that and remove hard coded. Thanks

**Comment by ACNHubert** [comment]
Thank you for your comments! :)

File: app/src/main/res/layout/list_item.xml (hunk 3 above)
Comment (ACNHubert) [reply app/src/main/res/layout/list_item.xml:11]: This file is already removed in my App

File: app/src/main/res/layout/list_item.xml (hunk 4 above)
Comment (ACNHubert) [reply app/src/main/res/layout/list_item.xml:17]: Same, This file is already remove in my App

**Pull request merged**